Automattic/woocommerce-payments
# Initial disputes list and evidence form

Fixes https://github.com/Automattic/woocommerce-payments/issues/279
Addresses https://github.com/Automattic/woocommerce-payments/issues/282 aside from charge-derived columns (including "Order #")

**Depends on 97-gh-woocommerce-payments-server**

* Initial "Disputes" list: <img width="1223" src="https://user-images.githubusercontent.com/1867547/69772279-7b827780-115d-11ea-8a3a-f11fc5bc51dd.png">

* Initial dispute evidence form, temporarily linked from "Needs response" status: <img width="1141" src="https://user-images.githubusercontent.com/1867547/69769400-ae733e00-1152-11ea-897e-26ba0d73a3a5.png">

Manages state in a thin wrapper around the view, rather than hooking into the existing data layer. Open both to fixing that in this PR and to leaving it as is for now.

Could also use some error handling.

#### Testing instructions

* Check out in test mode using a card from https://stripe.com/docs/testing#disputes
* Find the dispute in the Disputes list
* Click the icon next to the "Needs response" status to view evidence form
* Manipulate field(s), click "Save For Later", and observe that changes persist
* Manipulate field(s), click "Submit Evidence", and observe status change in Disputes list
    * As noted in above doc, use specific strings to trigger win or loss outcome

-------------------
